public/locales/en.json

```
@@ -114,6 +115,8 @@
     "label_option_vcenter_credential": "vCenter Credential",
     "label_option_ansible": "Ansible",
     "label_option_ansible_credential": "Ansible Credential",
+    "label_option_acs": "ACS",
+    "label_option_acs_credential": "ACS credential",
```

This should be capitalized, for consistency with other labels around:

```suggestion
    "label_option_acs_credential": "ACS Credential",
```